canvas_modules/common-canvas/src/common-properties/components/wide-flyout/wide-flyout.scss

```diff
@@ -24,12 +24,12 @@ $properties-modal-buttons-height: 64px;
 	left: 0;
 	right: 0;
 	bottom: 0;
-	background-color: $text-01;
+	background-color: $overlay;
```

Comment:
Shouldn't this be `$overlay-01`?

Reply:
@matthoward366  In Carbon10 `$overlay & $overlay-01` are the same color code. As per carbon11, the overlay color should be `$overlay` 
![image](https://github.com/elyra-ai/canvas/assets/151116685/e76ca59f-97e9-4e1e-87a8-cc555c2650ef)
![image](https://github.com/elyra-ai/canvas/assets/151116685/721ae276-f31b-4be0-ae26-15124979b57c)

Reply:
but we're on carbon 10 right now.

Reply:
Updated the code to use `$overlay-01`